src/components/LowerLayout/Footer/Footer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import OpenUpLogo from '../../../images/OpenUp-logo.png'
```

Comment:
Nice!

However, all variables in JavaScript should be https://en.wikipedia.org/wiki/Camel_case, even if they are proper nouns. Using Pascal case implies that the variable is a JavaScript class.